mdaencore/similarity.py

```diff
@@ -58,7 +58,6 @@
 
 .. bibliography::
     :filter: False
-    :style: MDA
```

Comment:
not shipping the MDA bib style?

Reply:
This wasn't done in the hole2 kit docs either, trying to keep it consistent between the two packages for when we create a proper extension

Reply:
ok, references in the hole kit look reasonable (although not as nice in the MDA docs) so I am fine with keeping it simple for right now. Maybe something for a docs extension.